extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -84,23 +84,6 @@ export function resetSharedState() {
   sharedState.reset();
 }
 
-/**
- * Mapping of experiment id to viewport offsets by which ad request should
- * be delayed.
- * @type {!Object<string, number>}
- * @visibleForTesting
- */
-export const DELAY_REQUEST_EXP_BRANCHES = {
-  '21062224': true, // control, delay by renderOutsideViewport
```

Comment:
Please remove all server side references to these experiments.

Reply:
will do